### PR TITLE
feat: Camera Service

### PR DESCRIPTION
**Pull request by jayfyle**
## Changes made:
1. Removed all imports for `@capacitor/camera` from components CaptureReceipt, CameraPreview
2.  Added the function `requestCameraPermission` which the only function used from the plugin

### CODE REVIEW

File: src/app/core/services/camera.service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { Camera, CameraPermissionType } from '@capacitor/camera';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CameraService {
+  constructor() {}
+
+  requestCameraPermissions(permissionsType?: CameraPermissionType[]) {
```
Comment (Julias0) [inline src/app/core/services/camera.service.ts:10]: we can remove the nullable here

File: src/app/core/services/camera.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { CameraService } from './camera.service';
+
+describe('CameraService', () => {
```
Comment (Julias0) [inline src/app/core/services/camera.service.spec.ts:5]: can we add unit tests for this

**Review by Julias0 — CHANGES_REQUESTED** [review]
Added comments

File: src/app/core/services/camera.service.ts (hunk 1 above)
Comment (jayfyle) [reply src/app/core/services/camera.service.ts:10]: It has to be optional as it defined as an optional in the plugin itself.  We are using this method twice, once with params and once without them.

**Review by Julias0 — APPROVED** [review]

**Pull request merged**